src/lib/LBT/LBT.cpp

```diff
@@ -0,0 +1,153 @@
+#include "common.h"
+#include "logging.h"
+#include "LBT.h"
+
+extern SX1280Driver Radio;
+
+LQCALC<100> LBTSuccessCalc;
+volatile uint32_t rxStartTime;
+volatile bool LBTChannelBusy = false;
+
```

Comment:
* rxStartTime does not need to be volatile, but should be static
* remove LBTChannelBusy entirely, use LBTSuccessCalc.current() to determined if channel was clear (no use having two variables storing the same thing)

Reply:
Implemented!